TKeazirian.HTTPServer/Response/ResponseBuilder.cs

```diff
@@ -21,16 +21,15 @@ public ResponseBuilder SetHeaders(string headerName, string headerValue)
         return this;
     }
 
-    public ResponseBuilder SetBody(string body)
+    public ResponseBuilder SetBodyBytes(byte[]? bodyBytes)
```

Comment:
I would expect to handle text as text and bytes as bytes. In other words, to extend this class.  Instead it appears that you have replaced one for the other.  How can you support both?